src/IRremoteESP8266.h

```diff
@@ -766,7 +773,7 @@
      DECODE_AMCOR || DECODE_DAIKIN152 || DECODE_MITSUBISHI136 || \
      DECODE_MITSUBISHI112 || DECODE_HITACHI_AC424 || DECODE_HITACHI_AC3 || \
      DECODE_HITACHI_AC344 || DECODE_CORONA_AC || DECODE_SANYO_AC || \
-     DECODE_VOLTAS || DECODE_MIRAGE || DECODE_HAIER_AC176)
+     DECODE_VOLTAS || DECODE_MIRAGE || DECODE_HAIER_AC176 || DECODE_TEKNOPOINT)
```

Comment:
This line looks a bit on the long end, even if it is ok now, should we have the foresight to add it on a new line?
Should we also end with a || \\nfalse to never have to care about that part ever changing?

Reply:
Oooo. I like the `|| false` idea. That's a great one. Will update it.

Reply:
Done.